src/drive/web/modules/views/Public/index.jsx

```diff
@@ -36,37 +29,12 @@ import { FabContext } from 'drive/lib/FabProvider'
 
 import usePublicFilesQuery from './usePublicFilesQuery'
 import usePublicWritePermissions from './usePublicWritePermissions'
-
-const getBreadcrumbPath = (t, displayedFolder, parentFolder) =>
-  uniqBy(
-    [
-      {
-        id: get(parentFolder, 'id'),
-        name: get(parentFolder, 'name')
-      },
-      {
-        id: displayedFolder.id,
-        name: displayedFolder.name
-      }
-    ],
-    'id'
-  )
-    .filter(({ id }) => Boolean(id))
-    .map(breadcrumb => ({
-      id: breadcrumb.id,
-      name: breadcrumb.name || '…'
-    }))
+import { ROOT_DIR_ID } from '../../../../constants/config'
```

Comment:
import { ROOT_DIR_ID } from 'drive/constants/config'


We need to talk about relative import. Can you add it to our next meeting ? ;)

Reply:
yes, next grooming